src/pages/admin/reports/components/pdf-files-section/components/pdf-section-content-block/PdfSectionContentBlock.module.scss

```
@@ -0,0 +1,73 @@
+    width: 100%;
+}
+
+.view-field {
+    display: flex;
+    flex-direction: column;
+    gap: 5px;
+}
+
+.view-label {
+    @include type.subtitle-1-bold;
+    color: c.$grey-900;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)